twake/backend/node/src/core/platform/framework/decorators/realtime/index.ts

```diff
@@ -1,14 +1,13 @@
 import { EntityTarget } from "../../api/crud-service";
 
-export * from "./created";
+export * from "./saved";
 export * from "./deleted";
-export * from "./updated";
 
 export interface PathResolver<T> {
   (type: T): string;
 }
 
-export function getPath<T>(path : string | PathResolver<T> = "/", entity: EntityTarget<T>): string {
+export function getPath<T>(path: string | PathResolver<T> = "/", entity: EntityTarget<T>): string {
```

Comment:
Avoid such changes which are useless in this scope. We will add prettier stuff and make a single commit for all the fixes

Reply:
We already all use prettier, I think you are the only one not using it @chamerling ;)

Reply:
VSCode is not happy with so much projects and config for now, will have to find the way to configure it correctly